recipes/nds2-client/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set name = "nds2-client" %}
+{% set version = "0.15.3" %}
+{% set sha256 = "b39616306cb0e5f80e052bedc34d2e93ac027d0f1c8ae3d2c813d9b60c5b3e81" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or not py2k]
+
+requirements:
+  build:
+    - make
+    - cmake >=3.6
+    - swig >=3.0.7
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - libcurl
+    - krb5
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - nds2
+  commands:
+    - nds-client-config --includedir --libdir
+    - nds_query --version
+
+outputs:
+  - name: nds2-client
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - libcurl
+        - krb5
+      run:
+        - libcurl
+        - krb5
+    files:
+      - bin/nds*
+      - etc/nds2-client/
+      - include/nds2-client
+      - share/man/man*/nds*
+      - lib/libnds*
+      - lib/pkgconfig/nds2-client.pc
+      - share/man/man*/nds*
+      - share/man/man*/NDS*
+    test:
+      commands:
+        - nds-client-config --includedir --libdir
+        - nds_query --version
+    about:
+      home: https://wiki.ligo.org/DASWG/NDSClient
+      dev_url: https://git.ligo.org/nds/nds2-client
+      doc_url: https://nds.docs.ligo.org/nds2-client/Beta/User/html/
+      license: GPLv3
+      license_family: GPL
+      license_file: COPYING
+      summary: NDS2 Client interface
+      description: |
+        The NDS2 client interface allows the user to down-load LIGO data
+        from V1 and V2 LIGO Network Data Servers.
+  - name: python-nds2-client
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - {{ pin_subpackage('nds2-client', exact=True) }}
+        - libcurl
+        - krb5
+        - python
+        - numpy
```

Comment:
Please, also change that to
```
- {{ pin_compatible('numpy') }}
```

Reply:
@marcelotrevisani, see 33c5499, unfortunately that broke the CI build...

Reply:
It's the opposite, sorry, the host is just ``numpy`` and to run you may ``pin`` it

Reply:
It may not be necessary, are you linking against numpy? If yes, you have o pin it, otherwise you can just let numpy and it may be good enough. I saw numpy in your build and I thought that you are linking against numpy

Reply:
@marcelotrevisani, I'm not 100% sure how this works, the build uses SWIG to provide python bindings of the code, but I'm not sure whether that means we have to pin...

Reply:
is what I need something like this:


```
requirements:
  ...
  host:
    ...
    - numpy {{ numpy }}
```

and then later in the sub-package output specs use `pin_compatbile('numpy')`?